### ISSUE TITLE
Emojicode operations unreadable with code block color

### ISSUE THREAD

**Issue by leios** [algorithm-archivists/algorithm-archive#624]
# Bug Report

## Description

The codeblock coloring renders emojicode operations (✖️, ➕, ✖️,◀️...) unreadable.

### Steps to Reproduce

1. Go to https://www.algorithm-archive.org/contents/monte_carlo_integration/monte_carlo_integration.html (or similar chapters) and click on emojicode.

### Expected behavior

To see the operators

## For Algorithm Archive Developers

- [ ] The bug can be reproduced
- [ ] The bug can be fixed (if not, please explain why not in a comment below)
- [ ] There is a timeline to fix the bug
- [ ] The bug has been fixed (Please link the PR)

**Comment by Liikt** [comment]
![Screenshot_20190722-123856_DuckDuckGo](https://user-images.githubusercontent.com/10891400/61626527-cc4d0580-ac7d-11e9-80cd-681b4c4703cc.png)

I can read them on my phone. I assume this has to do with the font you are using?

**Comment by leios** [comment]
Yeah, this certainly depends on the browser and font config. The question was what we should do about it. Should we change the bg? I don't know if it is possible to require a font for the AAA (because I don't understand the web)
